File: app/src/main/java/com/example/myanimedata/api/GenreResult.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class GenreResult {
    @SerializedName("mal_id")
    private int id;

    @SerializedName("name")
    private String name;

    @SerializedName("type")
    private String type;

    @SerializedName("url")
    private String url;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public String getUrl() {
        return url;
    }
}

File: app/src/main/java/com/example/myanimedata/api/GenreResult.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class GenreResult (
    @field:SerializedName("mal_id") val id: Int? = 0,
    @field:SerializedName("name") val name: String? = null,
    @field:SerializedName("type") val type: String? = null,
) : Parcelable